atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/KeyValueServices.java

```diff
@@ -208,7 +209,8 @@ public static RowColumnRangeIterator mergeGetRowsColumnRangeIntoSingleIterator(
         Map<byte[], RowColumnRangeIterator> rowsColumnRanges =
                 kvs.getRowsColumnRange(tableRef, rows, batchColumnRangeSelection, timestamp);
         // Return results in the same order as the provided rows.
-        Iterable<RowColumnRangeIterator> orderedRanges = Iterables.transform(rows, rowsColumnRanges::get);
+        Iterable<RowColumnRangeIterator> orderedRanges =
+                rows.stream().map(rowsColumnRanges::get).collect(Collectors.toList());
```

Comment:
This is allocation sensitive, may want to keep lazy transform

```suggestion
        Iterable<RowColumnRangeIterator> orderedRanges = Collections2.transform(rows, rowsColumnRanges::get);
```